Cargo.toml

```diff
@@ -81,6 +84,7 @@ targets = [
   "x86_64-unknown-linux-gnu",
   "x86_64-unknown-netbsd",
   "x86_64-unknown-openbsd",
+  "x86_64-unknown-hermit",
```

Comment:
Can you add this to the CI and the Makefile as well?

Reply:
Yes, I will do it.